srcpkgs/tmuxer/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'tmuxer'
+pkgname=tmuxer
+version=2.3.0
+revision=1
+archs=noarch
+depends="bash coreutils openssh sed tmux"
```

Comment:
i doubt it hard depends on openssh

Reply:
[It doesn't run without `ssh` being detected in $PATH]

See:

- https://github.com/tjhop/tmuxer/blob/master/tmuxer#L11
- https://github.com/tjhop/tmuxer/blob/master/tmuxer#L114